pkg/container/docker/network.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docker
+
+import (
+	"sigs.k8s.io/kind/pkg/exec"
+)
+
+// CreateNetwork create a bridge network
+func CreateNetwork(networkName string) error {
+	cmd := exec.Command(
+		"docker", "network",
+		"create",
+		"--driver=bridge",
+		networkName,
```

Comment:
we mentioned at same place [using labels for kind networks ](https://docs.docker.com/engine/reference/commandline/network_create/)

`docker network create ... --label k`

Reply:
sure, thanks. :+1:

Reply:
Addressed.